Xamarin.Forms.ControlGallery.WindowsUniversal/MainPage.xaml.cs

```diff
@@ -127,6 +129,30 @@ void AddNativeControls(NestedNativeControlGalleryPage page)
 				}
 			);
 
+			{
+				var stackLayout = new StackLayout()
+				{
+					VerticalOptions = LayoutOptions.Start,
+					HorizontalOptions = LayoutOptions.Center
+				};
+
+				var label = new Label
+				{
+					Text = "Hello World!",
+					VerticalOptions = LayoutOptions.Center,
+					HorizontalOptions = LayoutOptions.Center
+				};
+
+				var bz56843TextBlock = new TextBlock
```

Comment:
Since it doesn't successfully repro the problem, can we just remove the changes to this test? No point in making it more confusing than it already is.